app/controllers/admin/enterprises_controller.rb

```diff
@@ -11,6 +11,7 @@ class EnterprisesController < ResourceController
     before_filter :load_methods_and_fees, :only => [:edit, :update]
     before_filter :load_groups, :only => [:new, :edit, :update, :create]
     before_filter :load_taxons, :only => [:new, :edit, :update, :create]
+    before_filter :load_roles, only: :edit
```

Comment:
why don't you call this method from within `#edit`. This way the reader won't miss it when reading that method.